content/foundation/voting.md

```diff
@@ -117,10 +116,13 @@ for release, and there must be more positive than negative votes.
 Generally the community
 will cancel the release vote if anyone identifies serious problems, but
 in most cases the ultimate decision
-lies with the individual serving as release manager. The
+lies with the individual serving as release manager (RM). The
 specifics of the process may vary from project to project, but the 'minimum
 quorum of three +1 votes' rule is universal.
 
+Note that there is [no implicit +1](#no-implicit-1) from the RM.
```

Comment:
To avoid the need for the acronym you might write

> Note that there is [no implicit +1](#no-implicit-1) from the release manager, an explicit vote is required.

Reply:
Or even better, allowing for removing the "No implicit +1" section below:

> Note that there is [no implicit +1](https://github.com/apache/www-site/pull/338/files#no-implicit-1) from the release manager, or from anyone in any ASF vote. Only explicit votes are valid.